src/DynamoCore/Configuration/PathManager.cs

```diff
@@ -72,6 +72,7 @@ class PathManager : IPathManager
         private readonly string commonDataDir;
 
         private readonly string commonDefinitions;
+        private readonly string commonPackages;
```

Comment:
@saintentropy are you planning to add a TODO item to PathManager interface? or Add a new interface containing this item that we can merge back together later? This improves our chances of actually folding this back into any new apis in new releases.

Reply:
Will do